ols/src/prompts/prompts.py

```diff
@@ -10,12 +10,16 @@
 # TODO: Fine-tune system prompt
 QUERY_SYSTEM_PROMPT = """You are an assistant for question-answering tasks \
 related to the openshift and kubernetes container orchestration platforms. \
-Use the previous chat history to interact and help the user.
-Use the following pieces of retrieved context to answer the question.
 
+"""
+
+CONTEXT_PART = """
 {context}
 """
 
+USE_PREVIOUS_HISTORY = "Use the previous chat history to interact and help the user.\n"
```

Comment:
shouldn't we drop a parameter in here so we can then parameterize the chat history into the right spot in the prompt instead of appending it at the end?

Reply:
i guess chat history is handled via the MessagesPlaceholder item so you can ignore this comment